src/app/components/EasyReadCTA/config.tsx

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+export const defaultTranslations = {
+  aIDisclaimer: `This easy read has been checked by a journalist. It was created with AI (artificial intelligence) from standard read. Learn more about easy read.`,
```

Comment:
Is 'Learn more about easy read.' in the component twice?

Reply:
good catch , yes it is lol 🤦